### PR TITLE
Mention lazy loading for Doctrine event listeners

### PR DESCRIPTION
**Pull request by mpdude**
Would have saved me two hours if I had known about this feature before.

### CODE REVIEW

File: doctrine/event_listeners_subscribers.rst [hunk 1]
```diff
@@ -214,5 +214,60 @@ interface and have an event method for each event it subscribes to::
...
+        use AppBundle\EventListener\SearchIndexerSubscriber;
+
+        $container
+            ->register('my.listener', SearchIndexer::class)
+            ->addTag('doctrine.event_listener', array('event' => 'postPersist', 'lazy' => 'true'))
+        ;
+
+.. note::
+
+    Marking an event listener as ``lazy`` has nothing to do with lazy serivce
```
Comment (fsevestre) [inline doctrine/event_listeners_subscribers.rst]: Typo `serivce` -> `service`

**Review by xabbuh — APPROVED** [review]

File: doctrine/event_listeners_subscribers.rst [hunk 2]
```diff
@@ -214,5 +214,60 @@ interface and have an event method for each event it subscribes to::
...
+                    <tag name="doctrine.event_listener" event="postPersist" lazy="true" />
+                </service>
+            </services>
+        </container>
+
+    .. code-block:: php
+
+        use AppBundle\EventListener\SearchIndexer;
+        use AppBundle\EventListener\SearchIndexer2;
+        use AppBundle\EventListener\SearchIndexerSubscriber;
```
Comment (HeahDude) [inline doctrine/event_listeners_subscribers.rst]: It seems there's no need for those two last use statements.

**Review by HeahDude — APPROVED** [review]
Thanks for that PR!

File: doctrine/event_listeners_subscribers.rst (hunk 2 above)
Comment (mpdude) [inline doctrine/event_listeners_subscribers.rst]: @HeahDude Removed

**Comment by xabbuh** [comment]
Thank you @mpdude.

**Pull request closed without merging**